src/bench/gcs_filter.cpp

```diff
@@ -32,12 +32,65 @@ static void MatchGCSFilter(benchmark::Bench& bench)
         element[1] = static_cast<unsigned char>(i >> 8);
         elements.insert(std::move(element));
     }
-    GCSFilter filter({0, 0, 20, 1 << 20}, elements);
+    GCSFilter filter({0, 0, BASIC_FILTER_P, BASIC_FILTER_M}, elements);
 
     bench.unit("elem").run([&] {
         filter.Match(GCSFilter::Element());
     });
 }
 
+static void DecodeGCSFilter(benchmark::Bench& bench)
+{
+    GCSFilter::ElementSet elements;
+    for (int i = 0; i < 10000; ++i) {
+        GCSFilter::Element element(32);
+        element[0] = static_cast<unsigned char>(i);
+        element[1] = static_cast<unsigned char>(i >> 8);
+        elements.insert(std::move(element));
+    }
+    GCSFilter filter({0, 0, BASIC_FILTER_P, BASIC_FILTER_M}, elements);
+    auto encoded = filter.GetEncoded();
+
+    bench.unit("elem").run([&] {
+        GCSFilter filter({0, 0, BASIC_FILTER_P, BASIC_FILTER_M}, encoded);
+    });
+}
+
+static void BlockFilterGetHash(benchmark::Bench& bench)
```

Comment:
It would be handy to name all of these benchmarks with a common prefix to group them together in the output and that is easy to filter for, i.e. `./src/bench/bench_bitcoin -filter=EvictionProtection*.*`

Reply:
All the GCSFIlter benchmarks are now changed to have the prefix `GCSFilter`.